lib/proxy/startup_info.rb

```diff
@@ -0,0 +1,16 @@
+require 'proxy/request'
+require 'socket'
+
+module Proxy
+  class StartupInfo < Proxy::HttpRequest::ForemanRequest
+    include Proxy::Log
+
+    def put_features
+      params = { :proxy_name => Socket.gethostname, :startup_refresh => true }.to_json
```

Comment:
This may not return return expected (by foreman) hostname on a host with multiple interfaces.